src/proj.rs

```diff
@@ -86,11 +107,71 @@ fn area_set_bbox(parea: *mut proj_sys::PJ_AREA, new_area: Option<Area>) {
     }
 }
 
-/// A `PROJ` instance
-pub struct Proj {
-    c_proj: *mut PJconsts,
-    ctx: *mut PJ_CONTEXT,
-    area: Option<*mut PJ_AREA>,
+/// Enable or disable network access for [resource file download](https://proj.org/resource_files.html#where-are-proj-resource-files-looked-for).
+///
+/// This will configure network access for all **subsequent** `Proj` instances, but will **not** affect pre-existing instances.
+/// # Safety
+/// This method contains unsafe code.
+pub fn enable_network(enable: bool) -> Result<(), ProjError> {
+    if enable {
+        let _ = match set_network_callbacks() {
+            1 => Ok(1),
+            _ => Err(ProjError::Network),
+        }?;
+    }
+    let enable = if enable { 1 } else { 0 };
+    let dctx: *mut PJ_CONTEXT = ptr::null_mut();
+    unsafe { proj_context_set_enable_network(dctx, enable) };
```

Comment:
should we verify the return? 

https://proj.org/development/reference/functions.html#_CPPv431proj_context_set_enable_networkP10PJ_CONTEXTi

> returns TRUE if network access has been enabled

```
int proj_context_set_enable_network(PJ_CONTEXT *ctx, int enable) {
    if (ctx == nullptr) {
        ctx = pj_get_default_ctx();
    }
    // Load ini file, now so as to override its network settings
    pj_load_ini(ctx);
    ctx->networking.enabled_env_variable_checked = true;
    ctx->networking.enabled = enable != FALSE;
#ifdef CURL_ENABLED
    return ctx->networking.enabled;
#else
    return ctx->networking.enabled &&
           ctx->networking.open != NS_PROJ::no_op_network_open;
#endif
}
```

It seems almost but not quite redundant, e.g. if there were a situation where `ctx->networking.open != NS_PROJ::no_op_network_open` failed...

Reply:
Done!